run.sh

```diff
@@ -36,10 +36,14 @@ read sem
 echo -n "Введите номер задачи: "
 read zad
 
-curl -s  "https://mipt1.ru/1_2_3_4_5_kor.php?sem=$sem&zad=$zad" |  iconv -f WINDOWS-1251 -t UTF-8 > tmp
+if [ ! -f /tmp/gnu-koryavov/${sem}-${zad}.tmp ]; then
 
-status=$(egrep "Задача [[:digit:]]{1,2}\.[[:digit:]]{1,4} найдена" tmp)
-rm tmp
+    mkdir -p /tmp/gnu-koryavov/
```

Comment:
Maybe with sudo?

Reply:
This command can be executed by a non-privileged user on Debian and Ubuntu. Haven't tested on other systems
![Screenshot_20220402_143502](https://user-images.githubusercontent.com/89396128/161381273-22fb3a0e-19c2-4061-913b-bfef6b1f1582.png)

Reply:
Seems like `tmp` does not require elevated privileges.